### PR TITLE
Give sudo dialog more space

### LINKED ISSUE #2149: Administrative Access Required dialog is too narrow
### Actual Behavior

Fairly narrow dialog

### Steps to Reproduce

Upgrade to 1.3

### Result

image

### Expected Behavior

Wider dialog

### Additional Information

_No response_

### Rancher Desktop Version

1.3.0

### Rancher Desktop K8s Version

?

### Which container runtime are you using?

moby (docker cli)

### What operating system are you using?

macOS

### Operating System / Build Version

Big Sur 11.6.5

### What CPU architecture are you using?

x64

### Linux only: what package format did you use to install Rancher Desktop?

_No response_

### Windows User Only

_No response_

### PR DESCRIPTION
**Pull request by rak-phillip**
This gives the sudo dialog more space to display content by setting a min width and dialing back some margins and padding around contents.

### Before

Screen Shot 2022-05-05 at 2 36 54 PM

### After

Screen Shot 2022-05-05 at 2 33 47 PM

closes #2149

### CODE REVIEW

**Comment by jandubois** [comment]
I'll let somebody else review the CSS, but the new dialog looks good to me.

**Review by mook-as — APPROVED** [review]
Looks reasonable to me.

Some comments, though I think this might be preexisting and shouldn't block merging:
- In the `<details>`, the paragraph is right under the expanding-triangle-thing and looks a bit odd; it would probably look better with the left side flush against the line above (that is, "Provides bridge…" should be aligned to the line above, "Configure networking", both to the right of the ▼
- The spacing between paragraphs in the `<details>` is a bit tight (that is, the "Provides bridge…" paragraph vs the "This will modify…" paragraph).  Especially since the first paragraph ends at near the right margin for me.
-

**Comment by rak-phillip** [comment]
@mook-as I agree with your assessment, I created #2188 so we can focus on this change.

**Pull request merged**
